src/rpc/rpc.cpp

```diff
@@ -13,7 +13,7 @@
 c_rpc_server::c_rpc_server(const unsigned short port)
 :
 	m_io_service(),
-	m_acceptor(m_io_service, boost::asio::ip::tcp::endpoint(boost::asio::ip::address_v4::from_string("127.0.0.1"), port)),
+	m_acceptor(m_io_service, boost::asio::ip::tcp::endpoint(boost::asio::ip::address_v4::any(), port)),
```

Comment:
This needs to be a separate PR, that is marked as changing security;

For now do not do it, (except local tests, not committed here) - instead later add option to choose rpc bind adress.

Reply:
🆗